appdaemontestframework/appdaemon_mock/sccheduler.py

```diff
@@ -0,0 +1,148 @@
+
+import datetime
+import uuid
+import pytz
+from appdaemontestframework.appdaemon_mock.appdaemon import AppDaemon
+
+class Scheduler:
```

Comment:
Same as for appdaemon, would you mind renaming to `MockScheduler` to make more clear what this actually is. 🙂